src/test/cypress/support/requests/ExamManagementRequests.ts

```diff
@@ -0,0 +1,44 @@
+import { POST } from '../constants';
+import { CypressCredentials } from '../users';
+import exerciseGroup from '../../fixtures/requests/exerciseGroup_template.json';
+import { COURSE_BASE } from './CourseManagementRequests';
+
+/**
+ * A class which encapsulates all cypress requests related to managing an existing exam.
+ */
+export class ExamManagementRequests {
+    /**
+     * register the student for the exam
+     * @returns <Chainable> request response
+     */
+    registerStudent(course: any, exam: any, student: CypressCredentials) {
+        return cy.request({ method: POST, url: COURSE_BASE + course.id + '/exams/' + exam.id + '/students/' + student.username });
+    }
+
+    /**
+     * add exercise group to exam
+     * @returns <Chainable> request response
+     * */
+    addExerciseGroup(course: any, exam: any, title: string, mandatory: boolean) {
+        exerciseGroup.exam = exam;
+        exerciseGroup.title = title;
+        exerciseGroup.isMandatory = mandatory;
+        return cy.request({ method: POST, url: COURSE_BASE + course.id + '/exams/' + exam.id + '/exerciseGroups', body: exerciseGroup });
+    }
+
+    /**
+     * generate all missing individual exams
+     * @returns <Chainable> request response
+     */
+    generateMissingIndividualExams(course: any, exam: any) {
+        return cy.request({ method: POST, url: COURSE_BASE + course.id + '/exams/' + exam.id + '/generate-missing-student-exams' });
+    }
+
+    /**
+     * Prepares individual exercises for exam start
+     * @returns <Chainable> request response
+     */
+    prepareExerciseStart(course: any, exam: any) {
+        return cy.request({ method: POST, url: COURSE_BASE + course.id + '/exams/' + exam.id + '/student-exams/start-exercises' });
+    }
+}
```

Comment:
I think this whole class is a remnant from an earlier implementation of the exam participations branch. #3951 implements this separation of then in an extra PR. I think the class isn't used in your branch, so you can probably just delete this file without having any issues

Reply:
:+1: